### PR TITLE
Wip#11

### PR DESCRIPTION
**Pull request by andreasbalevik**
Added 'about' pages to Netlify CMS as suggested in #11

### CODE REVIEW

**Comment by andreasbalevik** [comment]
Some extra work is described in #14

**Comment by altinnadmin** [comment]
@andreasbalevik I think that we should keep on using frontmatter for ToC (`toc: true`) instead of requiring adding a shortcode. I assume Netlify supports editing frontmatter?

**Comment by andreasbalevik** [comment]
> @andreasbalevik I think that we should keep on using frontmatter for ToC (`toc: true`) instead of requiring adding a shortcode. I assume Netlify supports editing frontmatter?

Ok, added boolean for activating toc in Netlify CMS if needed. Updates value in front-matter.

File: README.md [hunk 1]
```diff
@@ -66,4 +66,10 @@ Workflow steps:
 4. Merge changes
 5. Published
 
+### Development
+The configuration for Netlify CMS has enabled `local_backen` which enables the CMS to read/store from local repository.
```
Comment (altinnadmin) [inline README.md:70]: Hmmm, ikke bra

Comment (altinnadmin) [inline README.md:70]: Missing d in `local_backen`?

File: content/om-altinn/driftsmeldinger/index.nb.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: Driftsmeldinger
 description: 
 didyoufind: false
 weight: 5
+cmsedit: true
+
```
Comment (altinnadmin) [inline content/om-altinn/driftsmeldinger/index.nb.md:7]: Empty line in frontmatter

File: content/om-altinn/personvern/index.en.md [hunk 3]
```diff
@@ -1,11 +1,12 @@
 ---
+lastmod: 2017-26
```
Comment (altinnadmin) [inline content/om-altinn/personvern/index.en.md:2]: Is this lastmod correct? (I know you have not changed it)

File: content/om-altinn/personvern/index.nb.md [hunk 4]
```diff
@@ -146,10 +168,11 @@ Dette gjør at analyseverktøyet kan anslå brukerens geografiske plassering, me
 Mottatte opplysninger er underlagt Google sine retningslinjer for personvern.
 
 ## Informasjonskapsler på altinn.no
+
 På altinn.no brukes det informasjonskapsler (cookies) for å bedre brukeropplevelsen av nettstedet, samt til statistikkformål.
 Her finner du en oversikt over de informasjonskaplsene vi bruker på altinn.no.
 
-**__RequestVerificationToken_**
+**_*RequestVerificationToken***
```
Comment (altinnadmin) [inline content/om-altinn/personvern/index.nb.md:175]: Is this correct? Ref. https://www.altinn.no/om-altinn/personvern/#informasjonskapsler-p%C3%A5-altinn-no

File: layouts/shortcodes/table-of-contents.html [hunk 5]
```diff
@@ -1,3 +1,5 @@
 <div>
+    <h2>{{ T "on-this-page" }}</h2>
     {{ .Page.TableOfContents }}
+    <br>
 </div>
```
Comment (altinnadmin) [inline layouts/shortcodes/table-of-contents.html:5]: Missing newline at end of file

**Review by altinnadmin — APPROVED** [review]
LGTM. Some minor comments for you to consider 👍

File: content/om-altinn/personvern/index.nb.md (hunk 4 above)
Comment (andreasbalevik) [reply content/om-altinn/personvern/index.nb.md]: Seems like markdown related issue. Changed to `**__RequestVerificationToken__**` and it outputs `RequestVerificationToken`. Hadde to escape ´\_´for it to work. Not sure how this works with Netlify CMS.

Comment (altinnadmin) [reply content/om-altinn/personvern/index.nb.md]: Had a look at the actual cookie name, and it seems like the current info on altinn.no is wrong 😆 
- altinn.no
   image
- Actual cookie
   image

@andreasbalevik I guess just escaping `_` using `\` would work fine also in Netlify?
Something like this: `**\_\_RequestVerificationToken**`

File: layouts/shortcodes/table-of-contents.html (hunk 5 above)
Comment (altinnadmin) [reply layouts/shortcodes/table-of-contents.html:5]: Just github complaining
image

But there are good reasons https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

File: content/om-altinn/personvern/index.nb.md (hunk 4 above)
Comment (andreasbalevik) [reply content/om-altinn/personvern/index.nb.md]: Netlify escapes automatically if needed. 

This one is not a issue. 
`**__RequestVerificationToken**`

But this one, with closing `__` netlify automatically escapes. 
`**\_\_RequestVerificationToken\_\_**`

**Pull request merged**